src/frontend/get-jira-configuration.ts

```diff
@@ -50,18 +52,26 @@ const formatDate = function (date) {
 	};
 };
 
-export const getFailedConnections = (installations, subscriptions) => {
-	return installations
+export const getFailedConnections = (
+	installations: FailedInstallations[] | typeof Installation[],
+	subscriptions: SubscriptionClass[]
+) => {
+	return (installations as FailedInstallations[])
 		.filter((response) => !!response.error)
 		.map((failedConnection) => {
```

Comment:
making this `(failedConnection:FailedInstallation)` should cast it automatically from this point on.

Reply:
If I don't do it before line 60 I get the error `Property 'error' does not exist on type 'FailedInstallations | Installation'.
  Property 'error' does not exist on type 'Installation'.`

Reply:
ah right that's because it doesn't know which ones is which at that point, which is fair.  You could cast it as `unknown` in the filter then `FailedInstallation` in the map